### PR TITLE
[Feature] Add new "modal" filter's display mode

### PR DESCRIPTION
**Pull request by ksn135**
Add to your list builder params ```filtersMode: modal```
config.yml
list of employees
Filters modal dialog box

### CODE REVIEW

**Comment by ksn135** [comment]
Disable "Reset" button when no filters set
Disable Reset button

**Comment by ksn135** [comment]
@loostro @sescandell  Please, review PR

File: Resources/templates/CommonAdmin/ListTemplate/ResultsBuilderTemplate.php.twig [hunk 1]
```diff
@@ -22,6 +22,20 @@
         <div class="box box-primary">
             <div class="box-header list-header list-nb-results">
                 {{- block('list_nbresults') -}}
+{% if builder.filtersMode == 'modal' %}
+    {{ echo_set('class', '') }}
+    {{ echo_if("app.session.get('\\\\" ~ bundle_name ~ '\\\\' ~ builder.ModelClass ~ "List\\\\Filters',{}) is empty") }}
```
Comment (sescandell) [inline Resources/templates/CommonAdmin/ListTemplate/ResultsBuilderTemplate.php.twig]: What about using something coming from the Controller whereas a test on the session here?

Just a question, don't know what's the better.

**Comment by sescandell** [comment]
Looks good!

Just one question about "how to test if filters are empty or not"

File: Resources/templates/CommonAdmin/ListTemplate/ResultsBuilderTemplate.php.twig (hunk 1 above)
Comment (ksn135) [inline Resources/templates/CommonAdmin/ListTemplate/ResultsBuilderTemplate.php.twig]: @sescandell well, it seems simpler for me to check for empty filter in twig template

**Comment by sescandell** [comment]
Ok for filters tests...

Could you just please rebase your PR because of merge of your other PR.

Thanks,

**Comment by ksn135** [comment]
@sescandell rebase done

**Comment by sescandell** [comment]
Merged, thanks

**Pull request merged**